pkg/ebpf/events/events.go

```diff
@@ -168,31 +195,45 @@ func capturePolicyEvents(ringbufferdata <-chan []byte, log logr.Logger, enableCl
 				var rb ringBufferDataV6_t
 				buf := bytes.NewBuffer(record)
 				if err := binary.Read(buf, binary.LittleEndian, &rb); err != nil {
-					log.Info("Failed to read from Ring buf", err)
+					log.Info("Failed to read from Ring buf", "error", err)
 					continue
 				}
 
 				protocol := utils.GetProtocol(int(rb.Protocol))
 				verdict := getVerdict(int(rb.Verdict))
-
+				if rb.Verdict == uint32(0) {
+					direction := "egress"
```

Comment:
Why do we flip the direction from egress to ingress here when updating the counter?